api/api.go

```diff
@@ -63,6 +63,8 @@ const (
 	SpecExportOptionsEmpty   = "empty_export_options"
 	SpecMountOptions         = "mount_options"
 	SpecCSIMountOptions      = "csi_mount_options"
+	SpecCSIRawBlock			 = "fadirectRawBlock"
```

Comment:
most spec options appear to be snake case - `fa_direct_raw_block`, let's stick to that convention

Reply:
this variable is named `SpecCSIRawBlock`, but the value seems to be specific to FA direct integration. 

Is the goal here to enable FA direct csi raw block integration via a SC parameter? If so, let's just name the spec name & value something like `SpecFADirectRawBlock`.

Reply:
will do, yea this is just to pass the raw_block flag through driver options down to the filter level so NSM knows its a raw block